manifests/repo.pp

```diff
@@ -8,7 +8,7 @@
       if $version == undef {
         fail("The parameter 'version' for 'postgresql::repo' is undefined. You must always define it when osfamily == Redhat or Linux")
       }
-      class { 'postgresql::repo::yum_postgresql_org': }
+      class { 'postgresql::repo::download_postgresql_org': }
```

Comment:
Hehe, maybe its just best to say 'redhat' or something of that nature, changing the class name each time the download URI changes, might get annoying. I say 'redhat', because it matches the case statement: https://github.com/puppetlabs/puppetlabs-postgresql/blob/master/manifests/repo.pp#L7 ... likewise apt, might be come debian.pp. Nit pick, really.